### PR TITLE
plan, ddl: add validate in checkCreateTableGrammar, to fail creating …

### PR DESCRIPTION
**Pull request by winkyao**
…an empty table

Create a empty name table should fail with error:

```
MySQL [test]> create table `` (c int);
ERROR 1103 (42000): Incorrect table name ''
```

but TiDB did not failed this query:

```
MySQL [test]> create table `` (c int);
Query OK, 0 rows affected (32.26 sec)
```

### CODE REVIEW

File: ddl/ddl.go [hunk 1]
```diff
@@ -110,6 +110,8 @@ var (
 	ErrInvalidOnUpdate = terror.ClassDDL.New(codeInvalidOnUpdate, "invalid ON UPDATE clause for the column")
 	// ErrTooLongIdent returns for too long name of database/table/column.
 	ErrTooLongIdent = terror.ClassDDL.New(codeTooLongIdent, "Identifier name too long")
+	// ErrWrongTableName return for wrong table name.
+	ErrWrongTableName = errWrongTableName
```
Comment (shenli) [inline ddl/ddl.go]: We could just rename errWrongTableName to ErrWrongTableName.

Comment (winkyao) [inline ddl/ddl.go]: OK

**Comment by shenli** [comment]
LGTM
@zimulala @tiancaiamao PTAL

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**